### ISSUE TITLE
Handle minified / unminified Kendo

### ISSUE THREAD

**Issue by JeroenVinke** [aurelia-ui-toolkits/aurelia-kendoui-bridge#550]
There are a few scenarios we have seen:

1. someone wants to use unminified Kendo files instead of minified (https://github.com/aurelia-ui-toolkits/aurelia-kendoui-bridge/blob/master/src/button/button.js#L6). Smart module loaders such as JSPM support path mappings that allow for this kind of thing but other module loaders make this difficult. Relevant issue: https://github.com/aurelia-ui-toolkits/aurelia-kendoui-bridge/issues/518#issuecomment-230658549
2. someone does not have access to the unminified Kendo source code. I believe that the bower endpoint of Kendo PRO does not provide source code, just Kendo PRO in minified form
3. you could be loading kendo.all.min.js as a script tag and don't want to load any Kendo files through the module loader at all

I believe the best situation is to have users load Kendo in their app (they can use whatever they like, minified/unminified, npm/bower/vendors) and the bridge does not load any Kendo files at all. It would work with globals already available before aurelia-kendoui-bridge gets initialized. This gives the user maximum control.

However, if we change this then it will be a pretty big breaking change because users that are now letting the bridge load Kendo will need to load Kendo themselves.

**Comment by adriatic** [comment]
> I believe the best situation is to have users load Kendo in their app (they can use whatever they like, minified/unminified, npm/bower/vendors) and the bridge does not load any Kendo files at all. It would work with globals already available before aurelia-kendoui-bridge gets initialized. This gives the user maximum control.

Good reasoning - and a breaking change, which could cause a stir in the support demand. The best solution (regardless of the effort) would be to ship two versions, where the version where the bridge loads kendo files says that it will soon be deprecated,

If you agree with that - I can take this task to free you just a liitle :-)

**Comment by JeroenVinke** [comment]
@adriatic that would be a nice solution yep

**Comment by adriatic** [comment]
Here is what I propose:
1. master will be the "old version" which loads kendo toolkit from the bridge. Here I am not sure how to make it to warn the user that it (bridge) will be replaced eventually - the user should be able to turn it off.

2. new branch (named no-kendo-sdk) will leave getting kendo sdk to user app and will be used to create the pre-release version on npm. There should also be an article in the catalog app explaining how to use this one.

**Comment by adriatic** [comment]
This whole situation is best described as a random luck. As I started to work on this, the first thing to do was to refresh my understanding of how is KendoUI Bridge put together. That in turn, gave me a few great ideas, not only how to solve this specific situation (by allowing the user app configure the bridge with respect to kendo sdk - I got a lot more.

I will share more as I get things to work :-)
